### PR TITLE
fix the line height

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes https://github.com/PolymerElements/paper-tooltip/issues/36

### CODE REVIEW

File: paper-tooltip.html [hunk 1]
```diff
@@ -70,6 +70,7 @@
         outline: none;
         @apply(--paper-font-common-base);
         font-size: 10px;
+        line-height: 10px;
```
Comment (frankiefu) [inline paper-tooltip.html]: We can also use `line-height: 1;` so if `font-size` changes `line-height` would have the same value.

Comment (notwaldorf) [inline paper-tooltip.html]: oh, neat! i didn't know that was a thing! thanks!

**Comment by notwaldorf** [comment]
@frankiefu assigned this to you now :)

**Comment by frankiefu** [comment]
LGTM :+1:

**Pull request merged**